public/app/plugins/datasource/jaeger/datasource.ts

```diff
@@ -52,6 +53,14 @@ export class JaegerDatasource extends DataSourceApi<JaegerQuery> {
       );
     }
 
+    if (target.queryType === 'upload') {
+      if (!this.uploadedJson) {
+        return of({ data: [emptyTraceDataFrame] });
```

Comment:
Can we return just empty array? If I just run query without uploading anything it will start showing `Trace` panel (empty but with header) which considering there is no data seems unnecessary.